src/DFA.cc

```diff
@@ -324,8 +322,7 @@ DFA_State* DFA_State_Cache::Lookup(const NFA_state_list& nfas, DigestStr* digest
 				{
 				*p++ = '0' + (char)(id % 10);
 				id /= 10;
-				}
```

Comment:
Is this correct here? Should we always be breaking before the while here?

Reply:
If we break before the `while`, should the `while` be at the level of the `do` or indented to where the brace is?

Reply:
After talking to Robin, I'm leaving this one alone.